pom.xml

```diff
@@ -2239,6 +2239,20 @@
             </properties>
         </profile>
 
+        <profile>
+            <id>spark-3.5</id>
+            <modules>
+                <module>extensions/spark/kyuubi-extension-spark-3-5</module>
+            </modules>
+            <properties>
+                <delta.version>2.4.0</delta.version>
+                <spark.version>3.5.0</spark.version>
+                <!-- TODO: Iceberg does not support 3.5 -->
```

Comment:
the 1.4.0 just released with spark 3.5 support

Reply:
Great, I will bump iceberg to 1.4.0